README.md

```diff
@@ -20,33 +20,42 @@ The simulation is for the initial phase of the epidemic in Slovakia, based on [S
 
 ## Building using Conan
 
-Conan is able to download the dependencies and compile the project. However, you still need OpenMP on your system, though that usually comes installed with the compiler (for Clang, you need to also install `libomp`).
+[Conan](https://conan.io) is able to download the dependencies and compile the project. However, you still need OpenMP on your system, though that usually comes installed with the compiler (for Clang, you need to also install `libomp`).
 
-```sh
-conan create -b gtest -b missing $PATH_TO_THIS_REPOSITORY
-conan install -g deploy 'COVID-19-simulation/1.0.0@'
-# Alternatively, adding '-o growth_type=exponential' creates a binary with simulated exponential growth.
-# The simulation binary will be in COVID-19-simulation/bin.
-```
-
-You can also use Conan to install `protoc`.
+If you've never used Conan, you need to configure it first.
 
 ```sh
-conan install -g deploy 'protoc_installer/[>=0]@bincrafters/stable'
-./protoc_installer/bin/protoc --decode ...  # More information on protoc usage below
+conan profile new default --detect
+conan profile update settings.compiler.libcxx=libstdc++11 default
 ```
 
+Once configured, you can create the package.
+```
+conan create .
+# Alternatively, adding '-o growth_type=exponential' creates a binary with simulated exponential growth.
+```
 
 ## Running and viewing protobuf files
 
 ```sh
-COVID-19-simulation/bin/simulation data/Slovakia.data
-# The simulation creates a proto file 'polynomial.sim' or 'exponential.sim', which can be examined
+conan install -g virtualrunenv 'COVID-19-simulation/1.0.0@'
+source activate_run.sh
```

Comment:
Where does `activate_run.sh` come from? Is it created by Conan?

Reply:
Yes https://docs.conan.io/en/latest/reference/generators/virtualrunenv.html#usage